src/util/asset/assetUtils.ts

```diff
@@ -27,3 +27,12 @@ export async function generateOptIntoAssetTxns({
     );
   }
 }
+
+/**
+ * @returns Array of given asset ids, bigger first
+ */
+export function sortAssetIds(asset1ID: number, asset2ID: number): number[] {
+  const assets = [asset1ID, asset2ID];
+
+  return [Math.max(...assets), Math.min(...assets)];
+}
```

Comment:
```suggestion
export function sortAssetIds(...assetIDs: number[]): number[] {
  return assetIDs.sort((a, b) => b - a);
}
```

In addition, we can also create `sortAssets` that I mentioned in this [PR](https://github.com/Hipo/tinyman-js-sdk/pull/5#discussion_r998652481). cc:@jamcry


```
export function sortAssets<Asset extends Pick<TinymanAnalyticsApiAsset, "id">>(...assets: Asset[]): Asset[] {
  return assets.sort((a, b) => Number(b.id) - Number(a.id));
}
```